### DISCUSSION TITLE
I have a problem with my pole figure plot... help me!~

### DISCUSSION THREAD

**Discussion by kchoi74** [mtex-toolbox/mtex#633] [Ask Anything] [answered]
Hello guys,
my name is David from South Korea.

I'd like to ask you guys how I can set the pole figure correctly.

My sample is aluminum alloys (Al-Mg system), and took EBSD file as .ang format.

My code is like below, and i also attach the pole figure, which shows very strangely.

Can someone help me correctly..?

Best,
David.

----------------------------------------------------------------------------------
%% Import Script for PoleFigure Data
%
% This script was automatically created by the import wizard. You should
% run the whoole script or parts of it in order to import your data. There
% is no problem in making any changes to this script.

%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = crystalSymmetry('m-3m', [4 4 4], 'mineral', 'Aluminum', 'color', [0 1 1]);

% specimen symmetry
SS = specimenSymmetry('mmm');

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outOfPlane');

%% Specify File Names

% path to files
pname = 'C:\Users\jk\Desktop\matlab\Preliminary Test\Raw data';

% which files to be imported
fname = [pname '\AlMg6V3_ES.ang'];

%% Specify Miller Indice

h = { ...
  Miller(1,1,0,CS),...
  };

%% Import the Data

% create a Pole Figure variable containing the data
pf = PoleFigure.load(fname,h,CS,SS,'interface','rigaku');
----------------------------------------------------------------------

![Wrong plot](https://user-images.githubusercontent.com/75606041/102313924-ded38780-3fb4-11eb-8bf7-355d092ed1d6.PNG)

**Comment by kilir** [comment]
Hi,
you should try to import your EBSD data as EBSD data, not as rigaku pole figure data.
Check EBSD in the import_wizard.
Does that help?
Cheers,
Rüdiger

**Reply by kchoi74** [reply]
Thanks for the reply!

By the way... then if I want to get a pole figure data, then which data format I can use?

Thanks.
David.

**Comment by ralfHielscher** [comment, marked as answer]
Import your data as EBSD data, i.e., using
```matlab
ebsd = EBSD.load('myFile.ang')
```
and then have a look at https://mtex-toolbox.github.io/EBSDOrientationPlots.html
for plotting pole figures from EBSD data, which is basically
```matlab
h = Miller(1,1,0,ebsd.CS)
plotPDF(ebsd.orientations,h)
```


Ralf.

**Reply by kchoi74** [reply]
![aa](https://user-images.githubusercontent.com/75606041/102460061-cd0fe400-4089-11eb-9612-6d060a43c77d.PNG)

Seems better results. Thank you!
